### PR TITLE
CIV-9482 corrected cancel page and expert name label

### PR DESCRIPTION
**Pull request by bhagyashreesharma90**
### JIRA link ###
https://tools.hmcts.net/jira/browse/CIV-9482

### Change description ###
Update cancel page to show radio button inline

image

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/views/features/caseProgression/cancel-your-upload.njk [hunk 1]
```diff
@@ -32,7 +32,7 @@
           <form method="post" novalidate>
             {{ csrfProtection(csrf) }}
 
-            {{ yesNoRadioButton(t, 'option', 'option', '', form, '', {yesHtml: yesHtml, yes: 'COMMON.VARIATION_3.YES', no: 'COMMON.VARIATION_3.NO'}) }}
+            {{ yesNoRadioButton(t, 'option', 'option', '', form, '', {yesHtml: yesHtml, yes: 'COMMON.VARIATION_3.YES', no: 'COMMON.VARIATION_3.NO', classes: 'govuk-radios--inline'}) }}
```
Comment (LeonardoPalmeiro) [inline src/main/views/features/caseProgression/cancel-your-upload.njk:35]: why do we need this?

File: src/main/modules/i18n/locales/en.json [hunk 2]
```diff
@@ -403,7 +403,7 @@
       "CCJ_INSTALMENTS_OF": "Instalments of",
       "CCJ_INSTALMENTS_PAYMENT_FREQUENCY": "How often you want them to make this payment?",
       "CCJ_TOTAL_TO_BE_PAID": "Total to be paid by defendant",
-      "CCJ_WHEN_DO_YOU_WANT_TO_BE_PAID_BY": "When you want the defendant to pay?",   
+      "CCJ_WHEN_DO_YOU_WANT_TO_BE_PAID_BY": "When you want the defendant to pay?",
```
Comment (LeonardoPalmeiro) [inline src/main/modules/i18n/locales/en.json:406]: why do we have changes in this part?

**Review by LeonardoPalmeiro — CHANGES_REQUESTED** [review]

File: src/main/views/features/caseProgression/cancel-your-upload.njk (hunk 1 above)
Comment (bhagyashreesharma90) [reply src/main/views/features/caseProgression/cancel-your-upload.njk:35]: This we need to make both Yes and No radio button in one line.

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (bhagyashreesharma90) [reply src/main/modules/i18n/locales/en.json]: > why do we have changes in this part?

reverted this. this spaces were removed automatically when I saved file from IDE.

**Review by LeonardoPalmeiro — APPROVED** [review]
LGTM

File: src/main/modules/i18n/locales/en.json (hunk 2 above)
Comment (LeonardoPalmeiro) [reply src/main/modules/i18n/locales/en.json]: Cool, thanks

**Comment by bhagyashreesharma90** [comment]
Expert's name changes need to be reverted as per latest update on the ticket. https://tools.hmcts.net/jira/browse/CIV-9482

**Review by LeonardoPalmeiro — APPROVED** [review]

**Review by kurt-uk — APPROVED** [review]

**Pull request merged**